lib/CMakeLists.txt

```
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(tree-sitter
+        VERSION "0.22.6"
```

```suggestion
        VERSION "0.23.0"
```
(unless you have pre-release versioning -- which you should consider)